docs/javascript.md

```diff
@@ -0,0 +1,15 @@
+# JavaScript
+
+RapidRails leverages modern JS libraries and frameworks such as Importmaps, Alpine.js and Hotwire. Here's a brief overview of how each is used:
+
+## Importmaps
+
+Importmaps allow us to control how JavaScript modules are imported in our application. In this project, the configuration for import maps is found in [config/importmap.rb](https://github.com/danielpaul/RapidRails/blob/main/config/importmap.rb). This file contains the mapping of module specifiers to URLs. To find out more see the [importmaps-rails Github](https://github.com/rails/importmap-rails).
+
+## Alpine.js
+
+Alpine.js is a minimal JavaScript framework used for adding interactivity to the website. To find out more see the [Alpine Docs](https://alpinejs.dev/start-here).
```

Comment:
give examples of where we use them in the code and why they are useful for small things where having a custom stimulus controller might not make sense.